public/checkout.js

```diff
@@ -171,6 +171,10 @@ var packeteryLoadCheckout = function( $, settings ) {
 			var _hasPickupPoints = hasPickupPoints( carrierRateId ),
 				_hasHomeDelivery = !_hasPickupPoints;
 
+			if (_hasPickupPoints && settings.widgetAutoOpen && $('iframe#packeta-widget').length === 0) {
+				$('.packeta-widget-button').click();
```

Comment:
Co se stane, když způsob přepravy bude předvolen nebo zvolen cizí logikou? Na řádku 199 vidím volání funkce při inicializaci pokladní logiky. Nemůže se stát to, že widget se otevře při zobrazení pokladny? Je to žádoucí? Neměl by na to být eshopista upozorněn? Prosím ověřit, že toto zadavatel chce.

Reply:
Ano, když v košíku vybereš VM, po přechodu do pokladny se hned otevře widget. Podobně, jako kdyby to byla jediná dostupná metoda. Ověřím.

Reply:
S Jardou jsme to probrali, na jiných eshopech se to takto chová a zatím to tak necháme. Případně budeme řešit v jiném ticketu.